docs/Upgrade-Project-From-V1.md

```diff
@@ -1,20 +1,20 @@
-# Upgrade Guide For ask-cli beta (askx)
- 
-> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI Beta (askx)**
- 
-Skills created with CLI 1.x will need to update their project structure in order to deploy with the CLI Beta (askx). To help projects upgrade to the new format, we have provided an `askx util upgrade-project` command that will attempt to perform the necessary changes.  
- 
+# Upgrade Guide For ask-cli (ask)
+
+> **Use this guide to upgrade CLI 1.x projects for deployment with the CLI (ask)**
```

Comment:
I think the previous beta should be replaced by "v2" as this doc is to upgrade project from v1 to v2. This place (ask) should actually be `with the CLI v2`